api/src/Blocks.ts

```diff
@@ -96,7 +96,7 @@ export class GearBlock {
    */
   async getEvents(blockHash: `0x${string}` | Uint8Array): Promise<Vec<FrameSystemEventRecord>> {
     const apiAt = await this.api.at(blockHash);
-    return await apiAt.query.system.events();
+    return (await apiAt.query.system.events()) as Vec<FrameSystemEventRecord>;
```

Comment:
```suggestion
    return apiAt.query.system.events();
```
`return await` [is redundant](https://eslint.org/docs/rules/no-return-await).
Also is there a need to cast `Vec<FrameSystemEventRecord>` type manually? `apiAt.query.system.events()` already returns `Promise<Vec<FrameSystemEventRecord>>` as far as I see